<launch>
    <node pkg="orb_slam2_ros" type="rgbd" name="orb_rgbd_node" output="screen">
        <param name="vocabulary_path" value="$(find orb_slam2_core)/vocabulary/ORBvoc.txt"/>
        <param name="settings_path" value="$(find orb_slam2_core)/config/EuRoC.yaml"/>
    </node>
</launch>